Traceback (most recent call last):
  File "/Users/matteo/anaconda3/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/matteo/anaconda3/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matteo/anaconda3/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matteo/anaconda3/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/matteo/anaconda3/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Users/matteo/anaconda3/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/matteo/anaconda3/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
y1=np.linspace(0, 1, 20) # [-]
y2=1-y1

Y1, T = np.meshgrid(y1, Temperature) 

Bmix=np.zeros((np.size(Temperature),np.size(y1)))

figure=plt.figure(figsize=(12, 10))
axes = figure.gca(projection ='3d') 

for i in np.arange(0,np.size(Temperature)):
        Bmix[i,:]=B[i,0]*Y1[i,:]*Y1[i,:]+B[i,1]*(1-Y1[i,:])*(1-Y1[i,:])+B[i,2]*Y1[i,:]*(1-Y1[i,:])

surf=axes.plot_surface(Y1,T,Bmix,cmap=cm.coolwarm,
                       linewidth=0, antialiased=False,alpha=0.5)   

axes.view_init(25, -60)
plt.draw()

axes.set_xlabel('$y$', fontsize=14);
axes.set_ylabel('$Temperature, (K)$',fontsize=14);
axes.set_zlabel('B$_{mix}$, dm$^3$ mol$^{-1}$',fontsize=14);     

figure.colorbar(surf, shrink=0.5, aspect=10);
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[4], line 9[0m
[1;32m      6[0m Bmix[38;5;241m=[39mnp[38;5;241m.[39mzeros((np[38;5;241m.[39msize(Temperature),np[38;5;241m.[39msize(y1)))
[1;32m      8[0m figure[38;5;241m=[39mplt[38;5;241m.[39mfigure(figsize[38;5;241m=[39m([38;5;241m12[39m, [38;5;241m10[39m))
[0;32m----> 9[0m axes [38;5;241m=[39m [43mfigure[49m[38;5;241;43m.[39;49m[43mgca[49m[43m([49m[43mprojection[49m[43m [49m[38;5;241;43m=[39;49m[38;5;124;43m'[39;49m[38;5;124;43m3d[39;49m[38;5;124;43m'[39;49m[43m)[49m 
[1;32m     11[0m [38;5;28;01mfor[39;00m i [38;5;129;01min[39;00m np[38;5;241m.[39marange([38;5;241m0[39m,np[38;5;241m.[39msize(Temperature)):
[1;32m     12[0m         Bmix[i,:][38;5;241m=[39mB[i,[38;5;241m0[39m][38;5;241m*[39mY1[i,:][38;5;241m*[39mY1[i,:][38;5;241m+[39mB[i,[38;5;241m1[39m][38;5;241m*[39m([38;5;241m1[39m[38;5;241m-[39mY1[i,:])[38;5;241m*[39m([38;5;241m1[39m[38;5;241m-[39mY1[i,:])[38;5;241m+[39mB[i,[38;5;241m2[39m][38;5;241m*[39mY1[i,:][38;5;241m*[39m([38;5;241m1[39m[38;5;241m-[39mY1[i,:])

[0;31mTypeError[0m: FigureBase.gca() got an unexpected keyword argument 'projection'

